### PR TITLE
Update eus-eus-update.adoc

### PR DESCRIPTION
**Pull request by nikhilprajapati-world**
Added * xref:../../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[Updating a cluster that includes RHEL compute machines] at line 47

Version(s):

OCP 4.14+  EUS to EUS 

Issue:

Link to docs preview:

https://75653--ocpdocs-pr.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/eus-eus-update.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by nikhilprajapati-world** [comment]
@skopacz1 : Can you do the needful here.

**Review by skopacz1 — COMMENTED** [review]
Left one comment about the correct filepath for the link, and another suggestion to add the note about RHEL workers in this PR instead of your original one. Also, since the filepath of that link will be different for 4.12 and 4.13, I'd suggest applying this PR to 4.14+, otherwise you'll run into issues when getting this PR merged.

Let me know if you have any questions!

File: updating/updating_a_cluster/eus-eus-update.adoc
[the remark this answers is not in the document]
Comment (nikhilprajapati-world) [reply updating/updating_a_cluster/eus-eus-update.adoc:1]: Done. Did the changes.

**Comment by JianLi-RH** [comment]
hi @skopacz1  generally LGTM, but I have a question, can we escalate the important message into **Performing an EUS-to-EUS update** section? Since update RHEL workers is not a step of normal upgrade process, and we have same notification in both **EUS-to-EUS update using the web console**  and **EUS-to-EUS update using the CLI**, so maybe only only one notification is enough.

@shellyyang1989 @jiajliu @jianlinliu Please help confirm.

**Comment by JianLi-RH** [comment]
Thank you @skopacz1 , I am fine with 2 notes.

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**